libs/common/domain/src/lib/model/record/metadata.model.ts

```diff
@@ -84,6 +84,7 @@ export interface BaseRecord {
   extras?: Record<string, unknown>
   landingPage?: URL
   updateFrequency?: UpdateFrequency
+  status?: RecordStatus
```

Comment:
FYI `status` is already present in `DatasetRecord`.

Reply:
Yes but for some reason, `@Input() metadata: CatalogRecord` in `home-record` which is of type `ServiceRecord | DatasetRecord`, does not recognize `status`.

Reply:
There might be a reason !
The status only exists for Dataset records, so the solution is not to update the Parent type, but probably to cast the Record in the component.
If the record page is designed for dataset record, then its type should be `DatasetRecord` instead of `BaseRecord`

Reply:
Thanks @fgravin , it's changed :slightly_smiling_face: